site/docs/guidelines/tables-lists.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
+tags: ["Data Tables", "Tabular Data", "Cards", "Tiles", "Rows", "Grids"]
+inComparingSection: true
+---
```

Comment:
Are we going to add visuals to this? Would be great to embed an example of a list and an example of a table here.

Reply:
Yep, the embeds are tricky so future iteration